### PR TITLE
fix #55543 sfgate.com

### LINKED ISSUE #55543: sfgate.com
### Issue URL (Annoyance)
 [https://www.sfgate.com/news/article/Small-towns-across-the-Sierra-fear-tourists-will-15269265.php](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.sfgate.com%2Fnews%2Farticle%2FSmall-towns-across-the-Sierra-fear-tourists-will-15269265.php)
### Comment
Username: @00Buck00
> Want to get rid of Admiral notification. It’s on many sites. Also the stupid auto playing video in the bottom right. That’s also in many sites. I can block the elements with my own rules but they always come back in another browsing session. 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances, <br/>Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies<br/><br/><b>Other:</b><br/> AdGuard Safari
Other extensions used: | None

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55543

### CODE REVIEW

File: AnnoyancesFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -333,6 +333,7 @@ forbes.com#@#.AD-label300x250
 forbes.com#@#.AD-Rotate
 !
 mail.yahoo.com##div[aria-describedby="adblock-whitelist-cue"]
+||prepareplanes.com^$third-party
```
Comment (BlazDT) [inline AnnoyancesFilter/sections/antiadblock.txt:336]: Not requested when I visit the site.
Only seeing they check these elements (but not reproduced adblock message):
 cssClass: "pub_300x250 pub_300x250m pub_728x90 text-ad textAd text_ad text_ads text-ads text-ad-links"

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/antiadblock.txt:336]: Try to visit with american IP
<details>

</details>

Comment (Alex-302) [inline AnnoyancesFilter/sections/antiadblock.txt:336]: move close to anti-admiral rules

Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/antiadblock.txt:336]: somewhere under this rule?
`||courier-js.dallasnews.com/dist/courier.min.js`

Comment (Alex-302) [reply AnnoyancesFilter/sections/antiadblock.txt:336]: yep. Next time.

**Pull request merged**